src/applications/proxy-rewrite/README.md

```diff
@@ -1,80 +1,124 @@
+## Web components and TeamSites
+Previously, the VA.gov header/footer and the TeamSites header/footer shared quite a bit of code. This helped to reduce tech debt, code duplication and defects.
+
+However, an effort is underway to convert the VA.gov header/footer to web components in the Design System library. Because the code between the VA.gov header/footer and TeamSites header/footer is shared **and** TeamSites do not support web components, we needed to separate the code for Teamsites as much as possible.
+
+As a result, all of the TeamSites header/footer code is now contained in `src/applications/proxy-rewrite` and is 100% duplicative of the VA.gov header/footer. There is a single style file in `proxy-rewrite/sass`, and all of the injection code is still present within this folder. The application has very little dependency on other code within the vets-website repo (with the exception of Platform utilities).
+
+### The downsides of an encapsulated injected header/footer app
+Aside from tech debt and code duplication:
+
+1. Any visual or structural changes made to the VA.gov (modernized) header/footer should also be applied and tested in the injected header/footer to keep the experience consistent
+2. The injected header/footer uses some Platform utility functions. If there are changes to these utilities, you must remember to test those changes against the injected header/footer
+
+## Do I need content-build running locally to test TeamSites?
+**No, but:** `content-build` generates a static JSON file with all of the Drupal data for the header/footer that TeamSites uses. It is called `headerFooter.json` and lives in `vets-website/build/localhost/generated/`. In order to ensure you have the most up-to-date Drupal data, you will need to have built `content-build` very recently. But the `content-build` repository does not need to be running in order to load TeamSites locally as you will be hitting port 3001 (which does not work with `content-build`).
+
+This file is created by running `yarn build` from `content-build`. Without optimization, this build step can take upwards of 8 hours. [Refer to this guide](https://github.com/department-of-veterans-affairs/content-build?tab=readme-ov-file#optimizing-build-time) to optimize your `content-build` and more quickly get up and running. **You will only need to do this once**, or whenever Drupal data for the header/footer is updated and affects your testing. Otherwise, as long as the `headerFooter.json` file exists, you can test the injected header/footer locally.
+
+Note: [next-build](https://github.com/department-of-veterans-affairs/next-build) which will eventually replace `content-build` in its entirety but will pull Drupal data the same way.
+
+## Where does the header/footer data come from?
```

Comment:
There's some  duplication here with https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/header-footer/engineering#data--content, which you'll end up reviewing per our convo earlier. 

I'm wondering if it's necessary for this info to live in both places? Could we point to that doc, rather than spelling it all out in this one?

Reply:
Updated!